src/scores/processing/discretise.py

```diff
@@ -188,9 +188,8 @@ def proportion_exceeding(
     Args:
         data (xarray.Dataset or xarray.DataArray): The data from which
             to calculate the proportion exceeding `thresholds`
-        thresholds (iterable): The proportion of Flip-Flop Index results
+        thresholds (iterable): The proportion of results
```

Comment:
Consider

```suggestion
        thresholds (iterable): The proportion of values
```

and similar below